frontend/src/store/store.js

```diff
@@ -0,0 +1,20 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
```

Comment:
also we like to use the convention of 3rd party imports, blank line, imports from other libraries within your code base (you don't have any rn), blank line, then local imports (anything that starts with ./ or ../)
Please update all your js files to follow this convention